src/core_modules/capture-core/actions/navigateToEnrollmentOverview/navigateToEnrollmentOverview.actions.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import { actionCreator } from '../actions.utils';
+
+export const actionTypes = Object.freeze({
+    NAVIGATE_TO_ENROLLMENT_OVERVIEW: 'enrollmentNavigation.navigateToEnrollmentOverview',
+});
+
+type NavigateToEnrollmentOverviewProps = {|
```

Comment:
Hey, 
In order to maintain homogeneity with the rest of the folders structure, I think is more sensible to move the type declaration from `navigateToEnrollmentOverview.actions.js` file to `navigateToEnrollmentOverview.types.js`.
Thanks!

Reply:
I agree, thanks!